src/main/java/org/spongepowered/api/text/format/TextColors.java

```diff
@@ -43,7 +42,7 @@ private TextColors() {
      */
     public static final TextColor NONE = new TextColor() {
 
-        private final Color color = new Color(0, 0, 0, 0);
+        private final Color color = Color.WHITE;
```

Comment:
Ah an alpha of 0 makes it white, weird why this wasn't just java.awt.Color.WHITE to begin with